Small improvements to /app/readme (#8179)
nextcloud/all-in-one: app/readme.md

# AIO app for NextCloud

> This folder contains a NextCloud app, which will be automatically installed within the NextCloud instance.

Reviewer: Please ensure the correct typing of Nextcloud:
```suggestion
This folder contains a Nextcloud app, which will be automatically installed within the Nextcloud instance.
```

> It adds a link to the admin settings page that gives access to the All-in-One admin interface.

Reviewer: Usually I call it the AIO-interfaces in most other places:
```suggestion
It adds a link to the admin settings page that gives access to the AIO-interface.
```
